### PR TITLE
File structure

### PR DESCRIPTION
**Pull request by teelu**
Added basic structure for file layout

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bower_components/
+.DS_Store
```
Comment (h8rry) [inline .gitignore:3]: Always leave a line break at the end of each file

**Comment by h8rry** [comment]
Please fix your indentations for all files

**Comment by h8rry** [comment]
The rest looks good. Let me know here when things are fixed. Then we will merge and move onto the next PR

**Comment by teelu** [comment]
Should be better now.

**Comment by h8rry** [comment]
Thanks looks good. Let's move to #2

**Pull request merged**